packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -0,0 +1,416 @@
+
+```tsx example
+function Example() {
+  let [state, setState] = React.useState(false);
+
+  return (
+    <div>
+      <MenuTrigger onOpenChange={(isOpen) => setState(isOpen)}>
+        <ActionButton>
+            Menu Button
+        </ActionButton>
+        <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+          {item => <Item>{item.name}</Item>}
+        </Menu>
+      </MenuTrigger>
+      <span style={{'margin-left': '8px'}}>Current open state: {state.toString()}</span>
+    </div>
+  );
+}
+```
+
+Menu accepts an `onAction` handler which is triggered whenever a Menu item is selected.
+
+The example below uses `onAction` to tell the user which Menu item was selected.
+
+```tsx example
+function Example() {
+  let onAction = (value) => alert('Menu item selected: ' + value);
+
+  return (
+    <div>
+      <MenuTrigger>
+        <ActionButton>
+            Menu Button
+        </ActionButton>
+        <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name" onAction={onAction}>
+          {item => <Item>{item.name}</Item>}
+        </Menu>
+      </MenuTrigger>
+    </div>
+  );
+}
+```
+
+## Props
+
+### MenuTrigger
+
+<PropTable component={docs.exports.MenuTrigger} links={docs.links} />
+
+### Menu
+
+<PropTable component={docs.exports.Menu} links={docs.links} />
+
+## Visual Options
+
+### Align and Direction
+
+[View guidelines](https://spectrum.adobe.com/page/popovers/#Placement)
+
+```tsx example
+<MenuTrigger align="start" direction="bottom">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="start" direction="top">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="bottom">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+<MenuTrigger align="end" direction="top">
+  <ActionButton>
+      Menu Button
+  </ActionButton>
+  <Menu items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]} itemKey="name">
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
+```
+
+### Autofocus
+
+Applying `autoFocus` to the Menu of the MenuTrigger sets focus to the Menu upon opening.
+
+```tsx example
+<MenuTrigger closeOnSelect={false}>
+  <ActionButton>
+      Menu Button Autofocus
+  </ActionButton>
+  <Menu
+    items={[{name: 'Aardvark'}, {name: 'Kangaroo'}, {name: 'Snake'}]}
+    itemKey="name"
+    autoFocus>
+    {item => <Item>{item.name}</Item>}
+  </Menu>
+</MenuTrigger>
```

Comment:
Maybe change this example to be `autoFocus="first"`  since with`autoFocus="true"` it focuses the menu itself/the selected item which is hard to see in this example cuz there aren't any selected items. 

Might also be worthwhile to go into a bit more detail into what each autoFocus option does (true vs false vs "first" vs "last")
double check the following btw:
false means nothing gets autofocused regardless of selectedItem
true autofocuses the selectedItem/menu
first autofocuses the selectedItem/first item in the list
last autofocuses the selectedItem/last item in the list

Reply:
I just did examples. I didn't update the description.